iOSTest/Info.plist

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Launch Screens can now be done entirely within Info.plist without the need to maintain a storyboard.